### ISSUE TITLE
Can't figure out how to open accessible panel chat view on Linux

### ISSUE THREAD

**Issue by lramos15** [microsoft/vscode#188837]
Testing #188537

I hear that I should use `Alt + F1`, but cannot find anything in the command palette for executing this command. Alt + F1 on my Ubuntu box just brings up the windows selector. `Shift + Alt + F1` just toggles screen reader optimization and prompts me to restart

**Comment by meganrogge** [comment]
![Screenshot 2023-07-26 at 9 37 46 AM](https://github.com/microsoft/vscode/assets/29464607/947a162a-8a73-41ce-9521-899296b05f21)
Here's the command in the command palette. 

The bug is this iteration, we assigned `Shift + Alt + F1` to toggle accessibility mode, but that's what opens accessibility help on Linux since, as you suggested, `Alt + F1` won't work there.

**Comment by meganrogge** [comment]
@alexr00 are you using linux?

**Comment by meganrogge** [comment]
oh, I think I need to add `primary`

**Comment by lramos15** [comment]
<img width="834" alt="image" src="https://github.com/microsoft/vscode/assets/4544166/81feab0c-f09c-44e5-8bd9-cc89a49b0df8">
Verified

**Comment by alexr00** [comment]
Yes, I was using Ubuntu.
